README.md

```diff
@@ -1,7 +1,130 @@
-# smkit-ui-flutter-demo
+# SMKitUI SDK - Demo App
 
-Hold tight! This demo will soon be ready.
+### [Usage](#usage)
+* [Initialization](#initialization)
+* [Android](#Android)
 
-Can't wait? [contact us](mailto:support@sency.ai) and let us know.
+### [API](#usage)
```

Comment:
@SencyCaspit both links are to the same location #usage 
that's probably a mistake

Reply:
Fixed.